src/ray/rpc/client_call.h

```diff
@@ -270,6 +274,9 @@ class ClientCallManager {
   /// Get the main service of this rpc.
   instrumented_io_context &GetMainService() { return main_service_; }
 
+  friend class ray::GcsClientTest;
```

Comment:
What private things we are accessing?

Reply:
StampContext originally accessed cluster_id_ directly, but I've changed ClientCallManager to surface this in a public getter